packages/server/src/auth/enum/role.enum.ts

```diff
@@ -0,0 +1,5 @@
+export enum Role {
+  // number assignment should be base 2 value
+  user = 0,
```

Comment:
Very nitpicky, enum values are usually in upper camel case, so `User`, `Admin`, etc

Reply:
fixed